frontend/src/main/pages/Review/ReviewCreatePage.js

```diff
@@ -1,13 +1,51 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import ReviewForm from "main/components/Review/ReviewForm";
+import { Navigate } from 'react-router-dom'
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
 
 export default function ReviewCreatePage() {
+
+  const objectToAxiosParams = (review) => ({
+    url: "/api/reviews/post",
+    method: "POST",
+    params: {
+      itemID: review.itemID,
```

Comment:
`itemId: review.itemId`

this is causing issues with communicating between frontend and backend.